asmcli/commands/help.sh

```diff
@@ -36,7 +36,7 @@ help_subcommand() {
 }
 
 version_message() {
-  local VER; VER="${MAJOR}.${MINOR}.${POINT}-asm.${REV}+config${CONFIG_VER}";
```

Comment:
Changing it here is going to change it everywhere which will break a ton of things. Set a variable to $(version_message) before the kpt setter and then do `var="asmcli-${mess_var/+/.}"`` or something similar

Reply:
Done.